robotoff/models.py

```diff
@@ -74,7 +76,12 @@ class ProductInsight(BaseModel):
     # 1 = Validated
     annotation = peewee.IntegerField(null=True, index=True)
 
-    # If the insight was annotated manually, this field stores the username of the annotator.
+    # The number of votes for this annotation.
+    # Stored here for quick sorting.
```

Comment:
@zond @kulizhsy here is the SQL I'am planning to run to migrate:

```
ALTER TABLE product_insight ADD COLUMN IF NOT EXISTS n_votes integer NOT NULL DEFAULT 0;
```
I think the best is to run it ahead of deployment, as it should not disturb.

I am wondering, if I should also run a :
```
UPDATE product_insight SET n_votes=1 where annotation = 1;
```

Reply:
The migration SQL seems correct.

I don't think you need to run the other code - we'll just have to keep in mind that old annotations are annotated without votes. Setting `n_votes` without actually populating the vote table seems dangerous. Better to keep `n_votes` at zero and use it to identify old annotations without votes?

Reply:
I've run the migration on both preprod and actual prod (10.1.0.103)

cc: @ocervell